index.md

```diff
@@ -372,6 +372,17 @@ Here are some examples of messages and how the parameters would be represented a
 
 As these examples show, a trailing parameter (a final parameter with a preceding `':'`) has the same semantics as any other parameter, and MUST NOT be treated specially or stored separately once the `':'` is stripped.
 
+### Compatibility with incorrect software
+
+Servers SHOULD handle single `\r` or `\n` character as if it was a `\r\n` pair, to support existing clients that might send this. However, clients and servers alike MUST NOT send single `\r` or `\n` characters.
+
+Servers and clients SHOULD ignore empty lines.
+
+Servers SHOULD gracefully handle messages over the 512-bytes limit. They may:
+
+* Send an error numeric back.
```

Comment:
Is there a recommended numeric to send back?

Reply:
I don' think so

Reply:
I don't think so, besides ERR_UNKNOWNCOMMAND and ERR_NEEDMOREPARAMS.

Reply:
There is ERR_INPUTTOOLONG, but no idea how widespread that is.

Reply:
Hmm, ircu defined ERR_INPUTTOOLONG and IRCv3 reused it for message-tags, this looks like a good fit.

Reply:
iirc insp already has a response they can send back for this issue?